src/cdomains/writtenDomain.ml

```diff
@@ -0,0 +1,10 @@
+module VD = ValueDomain.Compound
+
+module AD = struct
+  include ValueDomain.AD
+  include Printable.Std
+end
```

Comment:
`Printable.Std` should always be included _first_ such that everything else can override those, instead of the dummy defaults overriding manual implementations:
```suggestion
module AD = struct
  include Printable.Std
  include ValueDomain.AD
end
```